frontend/common/src/lang/fr.json

```diff
@@ -107,11 +107,11 @@
     "description": "Label for the privacy link in the Footer."
   },
   "ZE77nf": {
-    "defaultMessage": "canada.ca",
+    "defaultMessage": "Canada.ca",
```

Comment:
This one is now identical in both.  I think it should be in the whitelist rather than the fr.json file.
![image](https://user-images.githubusercontent.com/8978655/175966835-9f9bcc94-f203-47de-9d83-11797e3e9a45.png)